qrscanner/build.gradle

```diff
@@ -2,6 +2,7 @@ apply from: '../buildsystem/android_library.gradle'
 apply from: '../buildsystem/constants.gradle'
 apply from: '../buildsystem/helpers.gradle'
 apply from: '../buildsystem/test_dependencies.gradle'
+apply from: '../buildsystem/coverageReport.gradle'
```

Comment:
Like I mentioned in the standup, not sure if having the QR Scanning being covered with tests is a priority, considering the OKRs' definition. Would be nice if @rmeissner could provide some feedback here.

Reply:
it is not considered for the OKR, but I would still generate the test coverage for it. In a "perfect" world we should have tests for it (e.g. to make sure that inverted QR codes work ;) )

Reply:
Should we change the format of the `coverageReport.gradle` file name to snake case? (seems that you guys prefer this for file names ;) )

Reply:
I agree with changing the casing too

Reply:
Done in 077d4e65041543446b06c1307ecf89d3dc4d2807